### PR TITLE
✨ Sequencing Experiment Resource

### PR DESCRIPTION
**Pull request by znatty22**
Sequencing experiment CRUD endpoints, unit tests, and docs

Addresses #190

### CODE REVIEW

File: dataservice/api/__init__.py [hunk 1]
```diff
@@ -18,7 +18,12 @@
 from dataservice.api.sample import SampleListAPI
 from dataservice.api.aliquot import AliquotAPI
 from dataservice.api.aliquot import AliquotListAPI
-
+from dataservice.api.sequencing_experiment.resources import (
```
Comment (dankolbman) [inline dataservice/api/__init__.py]: Can you export these in the `sequencing_experiment` `__init__.py` to shorten this up onto one line and be consistent with the other imports?

Comment (dankolbman) [reply dataservice/api/__init__.py]: EDIT I see you already did that. You could just get rid of the `.resources`

Comment (znatty22) [reply dataservice/api/__init__.py]: Ah woops, 👍

**Review by dankolbman — APPROVED** [review]
🎊

**Review by parimalak — APPROVED** [review]
👍

**Pull request merged**